packages/cyberstorm/src/components/LinkLibrary.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  LinkLibrary,
+  thunderstoreLinkProps,
+  ThunderstoreLinkProps,
+} from "@thunderstore/components";
+import NextLink from "next/link";
```

Comment:
As mentioned by Mythic in Discord, `@thunderstore/cyberstorm` should be agnostic about the framework using the components. And also note that `@thunderstore/cyberstorm-storybook` doesn't use Next.js either.

And you didn't add the dependency to packages.json either.

Reply:
This LinkLibrary file is now removed, since it is not what we want to have. All of the stuff imported from this file are now imported from the correct place. The new LinkingProvider file under cyberstorm.